### PR TITLE
Update experiments to easy retrieve priors

### PR DESCRIPTION
**Pull request by xjules**
- [x] get single experiment endpoint
- [x] migrate prior retrieval test into experiment test
- [x] remove prior endpoint
Links to #119

### CODE REVIEW

File: src/ert_storage/endpoints/experiments.py [hunk 1]
```diff
@@ -104,6 +124,35 @@ def delete_experiment(*, db: Session = Depends(get_db), experiment_id: UUID) ->
...
+    ds.PriorFunction.stdnormal: PriorStdNormal,
+    ds.PriorFunction.uniform: PriorUniform,
+    ds.PriorFunction.ert_duniform: PriorErtDUniform,
+    ds.PriorFunction.loguniform: PriorLogUniform,
+    ds.PriorFunction.ert_erf: PriorErtErf,
+    ds.PriorFunction.ert_derf: PriorErtDErf,
+}
+
+
+def prior_to_dict(prior: ds.Prior) -> dict:
```
Comment (oysteoh) [inline src/ert_storage/endpoints/experiments.py:142]: seems like this is an internal func - should it be renamed to `_prior_to_dict`?

**Review by oysteoh — DISMISSED** [review]
Looks good to me - seems to do what it supposed to do! Had one minor comment regarding naming of what seems like a private func.

File: src/ert_storage/endpoints/experiments.py (hunk 1 above)
Comment (xjules) [reply src/ert_storage/endpoints/experiments.py:142]: Sure, sounds good!

File: src/ert_storage/endpoints/experiments.py [hunk 2]
```diff
@@ -104,6 +124,35 @@ def delete_experiment(*, db: Session = Depends(get_db), experiment_id: UUID) ->
...
+    return (
+        PRIOR_FUNCTION_TO_PYDANTIC[prior.function]
+        .parse_obj(
+            {key: val for key, val in zip(prior.argument_names, prior.argument_values)}
+        )
+        .dict()
+    )
+
+
+def experiment_priors_to_dict(experiment: ds.Experiment) -> Mapping[str, dict]:
```
Comment (oysteoh) [inline src/ert_storage/endpoints/experiments.py:152]: Sorry, forgot to post this one, i guess this should be private as well? Everything except the functions annotated with `@router` i guess is meant being private?

Comment (xjules) [reply src/ert_storage/endpoints/experiments.py:152]: Also this's used in other places too.

**Review by oysteoh — APPROVED** [review]

**Pull request merged**